classes/task/delete_meeting_recordings.php

```diff
@@ -0,0 +1,110 @@
+<?php
+// This file is part of the Zoom plugin for Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * The task for deleting recordings in Moodle if removed from Zoom.
+ *
+ * @package    mod_zoom
+ * @author     Jwalit Shah <jwalitshah@catalyst-au.net>
+ * @copyright  2021 Jwalit Shah <jwalitshah@catalyst-au.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace mod_zoom\task;
+
+defined('MOODLE_INTERNAL') || die();
+
+require_once($CFG->dirroot . '/mod/zoom/locallib.php');
+
+/**
+ * Scheduled task to delete meeting recordings from Moodle.
+ */
+class delete_meeting_recordings extends \core\task\scheduled_task {
+    /**
+     * Returns name of task.
+     *
+     * @return string
+     */
+    public function get_name() {
+        return get_string('getmeetingrecordings', 'mod_zoom');
```

Comment:
As @sgrandh3 identified, this should probably be:
```suggestion
        return get_string('deletemeetingrecordings', 'mod_zoom');
```
and a new string should be added for `deletemeetingrecordings` in lang/en/zoom.php

Reply:
Thank you for the testing comments @sgrandh3 and the note about incorrect label. I have updated the lang string to resolve this issue.